barretenberg/cmake/arch.cmake

```
@@ -5,6 +5,7 @@ if(WASM)
     add_compile_options(-fno-exceptions -fno-slp-vectorize)
 endif()
 
-if(NOT WASM AND NOT APPLE AND NOT ARM)
-    add_compile_options(-march=skylake-avx512)
+if(NOT WASM AND NOT APPLE AND NOT ARM) 
+#    add_compile_options(-march=skylake-avx512) # Required for AVX-512 extensions
+    add_compile_options(-march=skylake)
```

```suggestion
    add_compile_options(-march=skylake)
```
Let's also remove the commented line since it'll match upstream 😄

---

Done!